documentation/source/models_export.md

```diff
@@ -0,0 +1,900 @@
+
+### Flat format
+
+A detection results returned as a single tensor of shape `[N, 7]`, where `N` is the number of detected objects in the entire batch. Each row in the tensor represents a single detection result and has the following format:
+
+`[batch_index, x1, y1, x2, y2, class score, class index]`
+
+When exporting a model with batch size of 1 (default mode) you can ignore the first column as all boxes will belong to the single sample. In case you export model with batch size > 1 you have to iterate over this array like so:
+
+```python
+for sample_index in export_result.batch_size:
+    detections_for_sample_i = flat_predictions[flat_predictions[:, 0] == sample_index]
+    for (x1, y1, x2, y2, class_score, class_index) in detections_for_sample_i:
+        class_index = int(class_index) # convert from float to int
+        # do something with the detection predictions
+```
+
+### Batch format
+
+A second supported format is so-called "batch". It matches with output format of TensorRT's NMS implementation. The return value in this case is tuple of 4 tensors:
+
+* `num_predictions` - [B, 1] - A number of predictions per sample
+* `pred_boxes` - [B, N, 4] - A coordinates of the predicted boxes in X1, Y1, X2, Y2 format
+* `pred_scores` - [B, N] - A scores of the predicted boxes
+* `pred_classes` - [B, N] - A class indices of the predicted boxes
+
+Here `B` corresponds to batch size and `N` is the maximum number of detected objects per image.
+
+
+
+
+```python
+from super_gradients.training.datasets.datasets_conf import COCO_DETECTION_CLASSES_LIST
+from super_gradients.training.utils.detection_utils import DetectionVisualization
+import matplotlib.pyplot as plt
+
+
+def show_predictions_from_batch_format(image, predictions):
+    num_predictions, pred_boxes, pred_scores, pred_classes = predictions
+
+    assert num_predictions.shape[0] == 1, "Only batch size of 1 is supported by this function"
+
+    num_predictions = int(num_predictions.item())
+    pred_boxes = pred_boxes[0, :num_predictions]
+    pred_scores = pred_scores[0, :num_predictions]
+    pred_classes = pred_classes[0, :num_predictions]
+
+    image = image.copy()
+    class_names = COCO_DETECTION_CLASSES_LIST
+    color_mapping = DetectionVisualization._generate_color_mapping(len(class_names))
+
+    for (x1, y1, x2, y2, class_score, class_index) in zip(pred_boxes[:, 0], pred_boxes[:, 1], pred_boxes[:, 2], pred_boxes[:, 3], pred_scores, pred_classes):
+        image = DetectionVisualization._draw_box_title(
+                    image_np=image,
+                    x1=int(x1),
+                    y1=int(y1),
+                    x2=int(x2),
+                    y2=int(y2),
+                    class_id=class_index,
+                    class_names=class_names,
+                    color_mapping=color_mapping,
+                    box_thickness=2,
+                    pred_conf=class_score,
+                )
```

Comment:
Here the example is a function that plots bbox, while for `Flat format` just above, we just iterate over the results. I think it would make more sense to show 2 equivalent pieces of code